codegen/arrai/sysl.arrai

```diff
@@ -7,6 +7,12 @@ let app = (
 
 let isPtr = \t t('opt')?:false && !({"sequence", "set"} & (t => .@));
 
+let evaluateJsonTag = \type \key
```

Comment:
This definition can just live in `svc_types.arrai`. It doesn't have any context outside this file.

Nit: Rename to `jsonTag` or similar to match declarative style.

Reply:
ok